images/nginx/install-app.sh

```diff
@@ -2,15 +2,10 @@
 set -e
 set -x
 
-APP=$1 BRANCH=$2 GIT_URL=$3
+APP=$1
 
 cd /frappe-bench
 
-if test "$BRANCH" && test "$GIT_URL"; then
-  # Clone in case not copied manually
-  git clone --depth 1 -b "$BRANCH" "$GIT_URL" "apps/$APP"
-fi
-
 # Add all not built assets
 cp -r "apps/$APP/$APP/public" "/out/assets/$APP"
```

Comment:
If app doesn't have `public` directory? can this command not fail with non 0 exit?

Reply:
https://github.com/frappe/frappe_docker/issues/735#issuecomment-1081641829

Reply:
> Users shouldn't even try to install apps without frontend code, or I am wrong here?

Some apps may not have frontend code. If you create new app it won't have frontend code. If you do bench get-app, it'll run through the script and build no assets. Yet it goes through the yarn commands. `install-app` should not fail for such apps.